buildSrc/build.gradle

```diff
@@ -128,7 +128,7 @@ dependencies {
   testFixturesApi "com.carrotsearch.randomizedtesting:randomizedtesting-runner:${props.getProperty('randomizedrunner')}"
   testFixturesApi gradleApi()
   testFixturesApi gradleTestKit()
-  testImplementation 'com.github.tomakehurst:wiremock-jre8-standalone:2.35.0'
```

Comment:
It probably should be updated to 2.35.1 since 3.0.x bundles Jakarta EE 9+ and Jetty 11